src/igvfd/schemas/measurement_set.json

```diff
@@ -120,6 +120,21 @@
             "type": "string",
             "format": "uri",
             "pattern": "^https?://github\\.com/IGVF/seqspec/blob/main/assays/\\S+/spec\\.yaml$"
+        },
+        "control_file_sets": {
+            "title": "Controls",
+            "description": "File sets that can serve as scientific controls for this measurement_set.",
+            "type": "array",
+            "minimum": 1,
+            "default": [],
```

Comment:
can not be default empty if minimum is 1 item

Reply:
done